neo/rawio/axonrawio.py

```diff
@@ -169,7 +174,17 @@ def _parse_header(self):
             if (fSynchTimeUnit != 0) and (mode == 1):
                 length /= fSynchTimeUnit
 
-            self._raw_signals[seg_index] = self._raw_data[pos : pos + length].reshape(-1, nbchannel)
+            # self._raw_signals[seg_index] = self._raw_data[pos : pos + length].reshape(-1, nbchannel)
+
+            self._buffer_descriptions[0][seg_index] = {}
+            self._buffer_descriptions[0][seg_index][buffer_id] = {
+                "type" : "raw",
+                "file_path" : str(self.filename),
+                "dtype" : str(sig_dtype),
```

Comment:
is this for jsonification? rather than store a np.dtype()?

Reply:
If a data layout is split with a structure theorically we could handle it in zarr giving the list of chunks.
This could be done a the futur but this will not be in the actual PR.

Reply:
str(sig_dtype) is for jsonification yes.

Reply:
There are some without this safety step. So if we really want to be safe we should do this for all uses of dtype and Path no?